schema.xml

```diff
@@ -484,7 +496,8 @@
         Longer patterns will be matched first.  if equal size patterns
         both match, the first appearing in the schema will be used.  -->
 
-   <dynamicField name="*_display" type="string" indexed="false" stored="true" multiValued="true" />
+   <dynamicField name="*_display" type="string" indexed="true" stored="true" multiValued="true" />
```

Comment:
*_display are supposed to not indexed by convention/our rule.  Could we keep that so... looks like *_ms is dynamic  string with store true and index ...